src/App.js

```diff
@@ -30,14 +30,17 @@ class App extends Component {
   handleSubmit(event) {
     if(!quotes.includes(this.state.value)){
       quotes.push(this.state.value);
-      this.setState({ quote: this.getQuote()});
+      this.renderNewQuote();
 
       console.log(this.state.quote )
```

Comment:
unless you're still using them for something, it's usually good practic to take your `console.log` debug statements out of code before you create a PR that'll then have them running on other devs' machines.  It's less of an issue these days, but not all browsers support `console` in the first place (many IEs don't).